assets/src/edit-story/app/font/actions/useLoadFontFiles.js

```diff
@@ -19,48 +19,45 @@
  */
 import { useCallback } from 'react';
 
-function useLoadFontFiles({ getFontByName }) {
+/**
+ * Internal dependencies
+ */
+import cleanForSlug from '../../../utils/cleanForSlug';
+import getGoogleFontURL from '../../../utils/getGoogleFontURL';
+
+function useLoadFontFiles() {
   /**
    * Adds a <link> element to the <head> for a given font in case there is none yet.
    *
    * Allows dynamically enqueuing font styles when needed.
    *
    * @param {string} name Font name.
    */
-  const maybeEnqueueFontStyle = useCallback(
-    (name) => {
-      if (!name) {
-        return;
-      }
+  const maybeEnqueueFontStyle = useCallback(({ family, service }) => {
```

Comment:
Heads up to @obetomuniz: we have the same issue with font loading: not all fonts are Google fonts. Some are system font and that may need a different loading logic. Though checks would likely be always the same.